clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -6056,8 +6056,19 @@ bool clang::CodeGen::noSystemDebugInfo(const Decl *D,
     if (isa<TypedefDecl>(D))
       return false;
 
-    // -fno-system-debug was used.  Do not generate debug info.
-    if (CGM.getCodeGenOpts().NoSystemDebug)
+#ifndef WINDOWS
+    // Generate debug for _M_insert_aux to allow debugging for
```

Comment:
It is not clear what is being attempted here. Can you please add a comment? Thanks

Reply:
Updated comment to explain why #ifndef WINDOWS is used.